include.xml

```diff
@@ -20,6 +20,8 @@
 	<define name="lime-html5" if="html5" />
 	<define name="lime-native" if="native" />
 	<define name="lime-openal" unless="lime-console || flash || html5" />
+	<define name="lime-mojoal" if="lime-openal ios || tvos || static_link" />
```

Comment:
I think that was meant to be:

<define name="lime-mojoal" if="lime-openal || ios || tvos || static_link" /

Reply:
It was supposed to be `if lime-openal AND ios OR tvos OR static_link` but I guess that wouldn't work. I tried to simplify it by simply swapping the two in a new commit